dbt_subprojects/daily_spellbook/models/xrpl/gas_xrpl_fees.sql

```diff
@@ -0,0 +1,169 @@
+{{
+    config(
+        schema = 'gas_xrpl',
+        alias = 'fees',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['block_date','unique_key'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')],
+        post_hook='{{ expose_spells(\'["xrpl"]\',
+                                    "sector",
+                                    "gas",
+                                    \'["krishhh"]\') }}'
+    )
+}}
+
+WITH xrp_prices AS (
+    SELECT
+        DATE_TRUNC('minute', minute) AS price_minute
+        ,AVG(price) AS price_usd
+    FROM {{ source('prices', 'usd') }}
+    WHERE symbol = 'XRP' 
+        AND blockchain IS NULL
+    GROUP BY 1
```

Comment:
```suggestion
    SELECT
        minute AS price_minute
        ,price AS price_usd
    FROM {{ source('prices', 'usd') }}
    WHERE symbol = 'XRP' 
        AND blockchain IS NULL
        {% if is_incremental() %}
        AND {{ incremental_predicate('minute') }}
        {% endif %}
```

Reply:
prices is already at minute level of granularity, no need to force it and aggregate price.
plz always filter prices incrementally, if doing the same on source data.